share/tools/web_config/js/colorutils.js

```diff
@@ -393,19 +393,32 @@ function master_color_for_color(color_str) {
 
 /* Given a color name, like 'normal' or 'red' or 'FF00F0', return an RGB color string (or empty string) */
 function interpret_color(str) {
-	str = str.toLowerCase()
-	if (str == 'black') return '000000'
-	if (str == 'red') return 'FF0000'
-	if (str == 'green') return '00FF00'
-	if (str == 'brown') return '725000'
-	if (str == 'yellow') return 'FFFF00'
-	if (str == 'blue') return '0000FF'
-	if (str == 'magenta') return 'FF00FF'
-	if (str == 'purple') return 'FF00FF'
-	if (str == 'cyan') return '00FFFF'
-	if (str == 'white') return 'FFFFFF'
-	if (str == 'normal') return ''
-	return str
+    str = str.toLowerCase();
+    if (str == 'black') return '000000';
+    if (str == 'red') return '800000';
```

Comment:
There are good arguments for matching another terminal or some tangible standard. However if we will ever do something like have "set_color red" apply a 24bit color code instead of one of these 16 colors, I think there are also good arguments for very carefully picking out a great set of preferred colors that work well on both dark and light terminals. 

In the case above though, I was just going by this table, except for the colors which I left in for the sake of compatibility at their prior values that do not actually exist as named colors in ANSI (brown, purple, grey).

http://www.calmar.ws/vim/256-xterm-24bit-rgb-color-chart.html